specification/cognitiveservices/data-plane/TranslatorText/Batch/preview/v1.0/TranslatorBatch.json

```diff
@@ -0,0 +1,1463 @@
+        "operationId": "Batch Documents Submit",
+        "consumes": [
+          "application/json",
+          "text/json",
+          "application/*+json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "body",
+            "name": "body",
+            "description": "request details",
+            "schema": {
+              "$ref": "#/definitions/JobSubmissionBatchRequest"
+            }
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Successful request and the batch request is created by the service.  The header Operation-Location will indicate a status url with the operation id.",
+            "headers": {
+              "Operation-Location": {
+                "description": "Location of the operation",
+                "type": "string",
+                "format": ""
+              }
+            }
+          },
+          "400": {
+            "description": "Invalid request.  Check input parameters",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseV2"
+            },
+            "examples": {
+              "application/json": "{\r\n  \"error\": {\r\n    \"code\": \"InvalidRequest\",\r\n    \"message\": \"Some argument is incorrect\",\r\n    \"target\": \"document\",\r\n    \"innerError\": {\r\n      \"code\": 400120,\r\n      \"error\": \"SASTokenInvalid\",\r\n      \"message\": \"SAS token for storage is invalid\"\r\n    }\r\n  }\r\n}"
+            }
+          },
+          "401": {
+            "description": "Unauthorized.  Please check your credentials",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseV2"
+            },
+            "examples": {
+              "application/json": "{\r\n  \"error\": {\r\n    \"code\": \"Unauthorized\",\r\n    \"message\": \"User is not authorized\",\r\n    \"target\": \"document\",\r\n    \"innerError\": {\r\n      \"code\": 401000,\r\n      \"error\": \"Unauthorized\",\r\n      \"message\": \"Operation is not authorized\"\r\n    }\r\n  }\r\n}"
+            }
+          },
+          "429": {
```

Comment:
A 429 response should have a `Retry-After` header. I *believe* APIM is adding it.